agent/README.md

```diff
@@ -1,4 +1,65 @@
-# Mona Agent
+# Mona UDS Agent
 
-Mona agent
-Todo(Tzach): Complete this
+### Mona-UDS-Agent docker image
+
+This docker image runs mona-uds-agent, built on top of FluentD Technology.
+
+##### You should use mona-uds-agent when:
+1. You signed up on https://www.monalabs.io/getstarted and have a private endpoint to send your data to Mona
+2. Your models run on a docker orchestration such as Kubernetes and you wish to send their outputs directly to Mona
+3. You want to send your data through Unix Domain Sockets
+
+###### How to run it
+
+The following environment variables are required:
+
+* MONA_USER_ID: your user id (uuid format)
+
+You can use the following YAML formatted file on Kubernetes. 
+UDS agents have to be on all nodes, that's why it's required to use DaemonSet
+
+```
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: mona-uds-agent
+spec:
+  selector:
+    matchLabels:
+      app: mona-uds-agent
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        app: mona-uds-agent
+    spec:
+      initContainers:
+        - name: uds-init
+          imagePullPolicy: Always
+          image: monalabs/mona-agent-uds-init:stable
+          command: [ "sh", "./init.sh" ]
+          volumeMounts:
+            - name: uds-mona-socket
+              mountPath: /var/run/mona
+      containers:
+        - name: mona-uds-agent
+          imagePullPolicy: Always
+          image: monalabs/mona-uds-agent:stable
+          env:
+            - name: MONA_USER_ID
+              value: '<YOUR_PRIVATE_MONA_USER_ID>'
+          resources:
+            limits:
+              cpu: '2'
```

Comment:
Can these limits be played with as the user wishes? 

I would expect to have some details on:
1. Which yml fields (if any) are expected to remain exactly the same or the agent to function properly
2. If there are yaml fields (if any) which must have specific value ranges (e.g., never provide less than 1 cpu for this...)
3. Which yaml fields (if any) are expected to change (e.g., "we use 2 cpus by default as limit, but feel free to change this between the values X and Y") - you gave the user id example for this - so maybe you thought of this already :)

Reply:
Added some more info under the YAML.